CODING.md

```diff
@@ -6,7 +6,11 @@ Developers should keep their Go tooling up to date with the latest stable versio
 
 ## Static code analysis
 
-Linting is done by using https://github.com/golangci/golangci-lint with configuration defined in the root of the project and also `go vet`. Executing `make lint vet` should pass without any warnings or errors.
```

Comment:
go vet was removed because it is now part of the linters enabled?

Reply:
`make vet` was removed years ago from the Makefile.  `govet` is part of  `golangci-lint` now.